browser/search.mjs

```diff
@@ -103,24 +113,26 @@ const search = (state, push) => {
   })
 
   const searchTokenItem = (token, index, tokens) => {
-    console.log(token, index, tokens)
     const count = issuesPerToken[token] || 0
 
     if (token === 'or') {
-      return html`<li class="issue-search-query-or-item">or</li>`
+      return html`<li class="">or</li>`
     }
 
-    return html`<li>
-      <button
-        class="issue-search-query-item"
-        title="Remove ${token}"
-        value="${token}"
-        @click="${partial(onTokenRemove, index)}"
-      >
-        ${tokenIcon(token[0])} ${quoteOnWhitespace(token)}
-        <span class="badge badge-primary">${count}</span>
-      </button>
-    </li>`
+    return [
```

Comment:
Why return a list here if its only ever containing one element?

Reply:
This array will contain two elements if the previous token was not an "or".